modules/nodes-nodes-garbage-collection-configuring.adoc

```diff
@@ -43,42 +56,54 @@ $ oc label machineconfigpool worker custom-kubelet=small-pods
 
 .Procedure
 
-. Configure container garbage collection:
-
-.. Create a Custom Resource (CR) for your configuration change.
+. Create a Custom Resource (CR) for your configuration change.
 +
 .Sample configuration for a container garbage collection CR:
 [source,yaml]
 ----
 apiVersion: machineconfiguration.openshift.io/v1
 kind: KubeletConfig
 metadata:
-  name: gc-container <1>
+  name: worker-kubeconfig <1>
 spec:
   machineConfigPoolSelector:
     matchLabels:
       custom-kubelet: small-pods <2>
   kubeletConfig: 
-    EvictionSoft: <3>
-    - memory.available<100Mi <4>
-    - nodefs.available<10%
-    - nodefs.inodesFree<5%
-    - imagefs.available<15%
-    - imagefs.inodesFree<10%
-    EvictionSoftGracePeriod: <5>
-    - memory.available=1m30s
-    - nodefs.available=1m30s
-    - nodefs.inodesFree=1m30s
-    - imagefs.available=1m30s
-    - imagefs.inodesFree=1m30s
+    evictionSoft: <3>
+    - memory.available: "<100Mi" <4>
+    - nodefs.available: "<10%"
+    - nodefs.inodesFree: "<5%"
+    - imagefs.available: "<15%"
+    - imagefs.inodesFree: "<10%"
```

Comment:
@mburke5678 we need to remove all `<` (less than) operator for evictionSoft list options as it is creates syntax error and kubelet fails to start. Please refer my example kubeletConfig yaml https://github.com/openshift/openshift-docs/pull/14453#issuecomment-493147977. Also we need to remove `-` (hyphen) from the list as this also does not work.

Reply:
@sunilcio Sorry. Should have just copied your example instead of trying to fix mine. Wanted to show the less than option, not knowing it doesn't work.